src/app/work-item/work-item-detail/work-item-detail.component.ts

```diff
@@ -156,19 +156,19 @@ export class WorkItemDetailComponent implements OnInit, AfterViewInit, OnDestroy
             if (this.headerEditable && typeof(this.title) !== 'undefined') {
               this.title.nativeElement.focus();
             }
-          }
```

Comment:
There was one PR (https://github.com/fabric8io/fabric8-planner/pull/1733) got merged to master that will fix this part.